cmake/externalFunc.cmake

```diff
@@ -1,8 +1,10 @@
 set( CMAKE_EXECUTE_PROCESS_COMMAND_ECHO STDOUT)
 
+include(ExternalInclude)
+
 macro(addExternalFolder NAME FOLDER )
     # External resources/repositories are downloaded and built at configuration stage
-    message(INFO "[addExternalFolder] process ${NAME} ${FOLDER}")
+    message(VERBOSE "[addExternalFolder] process ${NAME} ${FOLDER}")
```

Comment:
The `VERBOSE` message mode was introduced in cmake 3.15. But the cmake minimal version set in the main `CMakeLists.txt` is 3.13. On several `CMakeLists.txt`, the minimum version is even set to 3.6 ... Should be useful to have coherent cmake minimum version along al the project and to ensure functionalities used are supported.

Reply:
I changed it as INFO does not exists. I can use any other.
The other changes you require are out of the scope of this PR, and can be fixed in another PR

Reply:
`INFO` mode has never existed.
See cmake doc about message :
https://cmake.org/cmake/help/v3.13/command/message.html for cmake version 3.13
or
https://cmake.org/cmake/help/v3.15/command/message.html for cmake version 3.15
3.15 and later allow much more message modes

And I think that cmake-minimum version check and coherency, even if not the initial goal of this PR could be fixed in the PR as modifications will be only on some already modified files.